client/my-sites/sharing/style.scss

```diff
@@ -900,24 +900,24 @@
 	}
 }
 
-@include sharing-button-service( 'facebook', $color-facebook );
-@include sharing-button-service( 'twitter', $color-twitter );
+@include sharing-button-service( 'facebook', var( --color-facebook ));
```

Comment:
I think so, but just to confirm. Something like this? 

```suggestion
@include sharing-button-service( 'facebook', rgba(var( --color-facebook )));
```

That looks like a confusing number of brackets though and just doesn't look right, so just want to be sure.

Reply:
Sorry, I think it would've been more clear from my side if I provided a code example. What I meant is the following:

```scss
/* _color-schemes.scss */
--color-facebook: #{$color-facebook};
--color-facebook-rgb: #{hex-to-rgb( $color-facebook )};

// this file
@include sharing-button-service( 'facebook', var( --color-facebook-rgb ));
```

This is necessary because the `sharing-button-service` mixin is it passing to `rgba( ... )` here:

https://github.com/Automattic/wp-calypso/blob/2d50a28f61230d2d2a2ab71088241ab7a056fc05/client/my-sites/sharing/style.scss#L897-L899

and the above `rgba` function doesn't work with hex values

Reply:
Thanks, that helps a ton! Just one more question, since `$color-facebook` would otherwise be removed, should they use HEX codes instead?

`--color-facebook-rgb: #{hex-to-rgb( #39579a )};`

Reply:
I think that'd work. Only notice that we need both `--color-facebook` and `--color-facebook-rgb` so there's some repetition but I think that may be fine in this case.

Reply:
Thank you! I've added a commit which intends to address this. Do you know if that's the same case [here](https://github.com/Automattic/wp-calypso/blob/4db6d989bc343e75463224251004bdecdee8c344/client/my-sites/sharing/style.scss#L267) too?

Reply:
The color values are directly used in the mentioned mixin so we can use the normal variables (without the `-rgb` suffix) there.